.github/workflows/updatemessages.yml

```diff
@@ -69,7 +69,7 @@ jobs:
             continue
           fi
 
-          if git diff "$file" | tail -n +5 2>/dev/null | grep -qP '^[-+](?!"PO-Revision-Date:|"POT-Creation-Date:)'; then
+          if git diff origin/update-i18n -- "$file" | tail -n +5 2>/dev/null | grep -qP '^[-+](?!"PO-Revision-Date:|"POT-Creation-Date:)'; then
```

Comment:
The problem here is what happens after you merge the PR and the branch is deleted? I agree that this approach should solve the problem of daily updates, but better logic is needed to avoid that special case.

Reply:
Done. This should work.
```bash
$ echo $(git rev-parse --verify -q origin/update-i18n || echo "HEAD")
9e610dd9dd5f0c7d302c9939a3f698d95fb7c2d0
$ echo $(git rev-parse --verify -q origin/update-i18nn || echo "HEAD")
HEAD
```